test/test_owners.csv

```diff
@@ -455,6 +455,10 @@ k8s.io/kubernetes/cmd/kube-proxy/app,luxas,1
 k8s.io/kubernetes/cmd/kubeadm/app/cmd,davidopp,1
 k8s.io/kubernetes/cmd/kubeadm/app/images,saad-ali,1
 k8s.io/kubernetes/cmd/kubeadm/app/util,eparis,1
+k8s.io/kubernetes/cmd/kubeadm/app/cmd,vishh,1
+k8s.io/kubernetes/cmd/kubeadm/app/images,davidopp,1
+k8s.io/kubernetes/cmd/kubeadm/app/preflight,apprenda,0
```

Comment:
can an organization be here?
I'm not sure that it's possible.

Can you just add yourself @pires or is this autogenerated?